src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content.models;
+
+import uk.gov.hmcts.cmc.domain.models.Address;
+
+public class PartyDetailsContent {
+
+    private final String type;
+    private final String fullName;
+    private final boolean nameAmended;
+    private final String businessName;
+    private final String contactPerson;
+    private final Address address;
+    private final boolean addressAmended;
+    private final Address correspondenceAddress;
+    private final String dateOfBirth;
+    private final String email;
+
+    public PartyDetailsContent(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 10 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)